libnetwork/netavark/network.go

```diff
@@ -99,7 +108,9 @@ func NewNetworkInterface(conf *InitConfig) (types.ContainerNetwork, error) {
 
 	n := &netavarkNetwork{
 		networkConfigDir: conf.NetworkConfigDir,
+		networkRunDir:    conf.NetworkRunDir,
```

Comment:
can we create a sub directory for aardvark

Reply:
aardvark already does it.

Reply:
here: https://github.com/containers/netavark/blob/main/src/commands/setup.rs#L208